index.js

```diff
@@ -0,0 +1,98 @@
+import domSelectors from './modules/domSelectors.js';
+import Store from './modules/LocalStorage.js';
+import Book from './modules/Book.js';
+// import UI from './modules/UI.js';
+
+setInterval(() => {
+  document.getElementById('date').textContent = new Date();
+}, 1000);
+
+// Show bookList
+domSelectors.bookListView.classList.toggle('show');
+
+// NavBar Eventlisteners
+domSelectors.GoToBookList.addEventListener('click', () => {
+  domSelectors.bookListView.classList.add('show');
+  domSelectors.addBookView.classList.remove('show');
+  domSelectors.contactView.classList.remove('show');
+});
+
+domSelectors.GoToAddBook.addEventListener('click', () => {
+  domSelectors.addBookView.classList.add('show');
+  domSelectors.bookListView.classList.remove('show');
+  domSelectors.contactView.classList.remove('show');
+});
+
+domSelectors.GoToContact.addEventListener('click', () => {
+  domSelectors.contactView.classList.add('show');
+  domSelectors.addBookView.classList.remove('show');
+  domSelectors.bookListView.classList.remove('show');
+});
+
+// ------  UI CLASS ----- //
+
+class UI {
+  // Display the list of Books
+  static displayBooks() {
+    const books = Store.getBooks();
+    books.forEach((book) => UI.addBookToList(book));
+  }
+
+  // Add book to the List
+  static addBookToList(book) {
+    const list = document.querySelector('#book-list-table');
+    const row = document.createElement('tr');
+    row.innerHTML = `
+          <td id='title-decor'>${book.title}</td>
+          by
+          <td>${book.author}</td>
+          <td>
+            <button><a href='#' id='delete-btn' class='delete'>Delete</a></button>
+          </td>
+        `;
+    list.appendChild(row);
+  }
+
+  // Delete a book from the storage
+  static deleteBook(el) {
+    if (el.classList.contains('delete')) {
+      el.parentElement.parentElement.parentElement.remove();
+    }
+  }
+}
+
+// Event: Display Books
+document.addEventListener('DOMContentLoaded', UI.displayBooks);
+
+// Event: Add a Book
+document.querySelector('#book-form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  // Get form values
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  // Validate
+  if (title === '' || author === '') {
+    // Do Nothing
+  } else {
+    // Instatiate book
+    const book = new Book(title, author);
+    // Add Book to UI
+    UI.addBookToList(book);
+    // Add book to store
+    Store.addBook(book);
+    // Clear fields
```

Comment:
- [ ] (Optional) It would be nice to clear the input fields after submitting the form. I guess that is what UI.clearFields() is meant to do, but you did not declare/define what the function should do.
- [ ] (Optional) You could also display a small message that shows the book was added successfully or it automatically redirects the user to the list of book section after submitting.

Reply:
I have added a Clear Fields in UI class.